src/sass/_mixins.scss

```diff
@@ -17,15 +17,28 @@
   margin: 0;
 }
 
+// this mixin allows inserting inline-based components without concerns about vertical dimension
+@mixin components-container() {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
 @mixin hole() {
   &__hole {
-    display: flex;
-    align-items: center;
-    justify-content: center;
+    @include components-container();
     height: 100%;
+    justify-content: center;
   }
 }
 
+// this mixin remove additional space for ascenders/descenders
+// should be applied for inline-block elements with height < line-height
+@mixin remove-descenders() {
```

Comment:
Used to remove outstanding space for `inline-block` elements that are smaller than `rhythm(1)`.

Used instead of setting `vertical-align` or adding ghost element to parent container.

Can be deprecated as soon as containers get `display: flex`